tests/rptest/tests/raft_availability_test.py

```diff
@@ -580,5 +579,6 @@ def controller_available():
                     hosts=hosts,
                     check=lambda node_id: node_id != controller_id)
 
+        connection = self.ping_pong_warmed()
         for i in range(0, 128):
-            self._ping_pong()
+            connection.ping_pong(ELECTION_TIMEOUT * 2)
```

Comment:
Given that there is now an explicit warmup phase, the timeout on each subsequent pingpong should probably be much lower.